packages/neon/neon_notifications/lib/pages/main.dart

```diff
@@ -74,16 +80,16 @@ class _NotificationsMainPageState extends State<NotificationsMainPage> {
       ),
       leading: app != null
           ? app.buildIcon(
-              size: largeIconSize,
-            )
+        size: largeIconSize,
+      )
           : SizedBox.fromSize(
-              size: const Size.square(largeIconSize),
-              child: NeonCachedImage.url(
-                url: notification.icon!,
-                size: const Size.square(largeIconSize),
-                svgColor: Theme.of(context).colorScheme.primary,
-              ),
-            ),
+        size: const Size.square(largeIconSize),
+        child: NeonCachedImage.url(
+          url: notification.icon!,
+          size: const Size.square(largeIconSize),
+          svgColor: Theme.of(context).colorScheme.primary,
+        ),
+      ),
```

Comment:
This looks like random formatting changes. Please run `melos run format` and commit the result.